### PR TITLE
Fixes #239: Added Emoji Heatmap with no search query

### LINKED ISSUE #239: Emoji Heatmap with no search query
- Issue type: Feature request

### Short description
I am trying to implement emoji heatmap where when the app loads, the heatmap shows occurrences of all the existing emojis location on map without any search query.
 
### Screencast link of the work done till now
https://www.screencast.com/t/Revds3tyvc

### Screenshot
screenshot 198

**Problems am facing while implementing this app**
When the app starts loading as am sending queries to the search api (query input for search is from a json file) continuously, the server is sometimes showing error 503 (which is about request frequency being too high).

So how can i solve this?
Screenshot of issue:
screenshot 199

### Would you like to work on the issue?
Yes.

### PR DESCRIPTION
**Pull request by kavithaenair**
### Short description
Related to #239 
Demo link: https://kavithaenair.github.io/emojiHeatmap/

Screenshot for changes:
image

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by kavithaenair** [comment]
@djmgit @singhpratyush @vibhcool @Achint08 Please review

File: emojiHeatmap/index.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  <title>Loklak Heatmap</title>
+</head>
+<body>
+  <nav class="navbar navbar-default navbar-fixed-top">
+    <div class="container">
+      <div class="navbar-header">
+        <button type="button" class="navbar-toggle collapsed"
+          data-toggle="collapse" data-target="#navbar"
+          aria-expanded="false" aria-controls="navbar">
+        <span class="sr-only">Toggle navigation</span>
```
Comment (djmgit) [inline emojiHeatmap/index.html]: spans are not indented

File: emojiHeatmap/index.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+  <!-- Bootstrap JS -->
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"
+          integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS"
+          crossorigin="anonymous"></script>
+  <!-- Loklak Fetcher Client lib -->
+  <script src="../emojiHeatmapper/loklak-fetcher-client.js"></script>
+  <!-- Main script -->
+  <script src="./heatmap.js"></script>
+
```
Comment (djmgit) [inline emojiHeatmap/index.html]: empty line

**Comment by kavithaenair** [comment]
@djmgit made changes. Please review. The link updated for now.
image

File: emojiHeatmap/index.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    <h3 id="sentiment"></h3>
+    <div class="panel panel-default">
+      <div class="panel-body">
+        <div id="map"></div>
+      </div>
+    </div>
+    <footer>
+      <hr>
+      <div class="well">
+        <a href="http://loklak.org"><b>LokLak Public API</b></a>
```
Comment (vibhcool) [inline emojiHeatmap/index.html]: link to api.loklak.org ..

**Comment by kavithaenair** [comment]
@vibhcool Done.

**Pull request merged**